### ISSUE TITLE
Add support for service cron (feature request)

### ISSUE THREAD

**Issue by andrey-tech** [manticoresoftware/docker#42]
# Feature request

Please add support for service cron in Docker container of Manticore Search.

For today I need to create custom script for `ENTRYPOINT` to add all services missing in the Docker container (including crond), since internal logic of your script `docker-entrypoint.sh` is strictly tied to launch `searchd` or `indexer` from `CMD`.

I ask you to refactor `docker-entrypoint.sh` and add the ability to run my own script incudes all missing commands.

_Forum discussion (russian): https://forum.manticoresearch.com/t/crond-docker/1616_

**Comment by sanikolaev** [comment]
Hi. Please elaborate more on what you need cron inside the container for: regular indexing or smth else.

**Comment by andrey-tech** [comment]
@sanikolaev 
Hello! I need cron inside the container for regular indexing, e.g. every hour from 08:00 to 20:00.

**Comment by sanikolaev** [comment]
OK, in this case we can try to make things simpler for the user than just enabling crond inside the container and letting user modify `/etc/crontab` or such, e.g.:
- we already support `CREATE_PLAIN_TABLES=1` flag to build plain tables once on starting the container
- in addition to that we can:
  - add support for `CREATE_PLAIN_TABLES=tbl;tbl2;tbl3` to build only specified tables on start
  - add support for e.g. `CREATE_PLAIN_TABLES="tbl:* * * * *;tbl2:*/5 2 * * *"` to:
    - build table `tbl` every minute
    - build table `tbl2` every 5th minute of 2am
  - run indexing through a lock to make sure we don't run another indexing of the same type if the previous one is not complete yet
  - keep indexing logs in `/var/log/manticore/indexing_tbl.log`, `/var/log/manticore/indexing_tbl2.log` etc. with info on when each indexing started and finished

**Comment by andrey-tech** [comment]
OK. Please, don't forget to add the `--rotate` flag for `indexer`.

**Comment by djklim87** [comment]
Done in 890663a4b2b8668176201ce20f1c838ead896972

**Comment by sanikolaev** [comment]
@PavelShilin89 pls test this new feature and cover with a CLT test.

**Comment by PavelShilin89** [comment]
@djklim87 Running crontab fails with this error: 
```
Manticore 6.2.13 60343e9d9@231017 dev
Copyright (c) 2001-2016, Andrew Aksyonoff
Copyright (c) 2008-2016, Sphinx Technologies Inc (http://sphinxsearch.com)
Copyright (c) 2017-2023, Manticore Software LTD (https://manticoresearch.com)

using config file '/etc/manticoresearch/manticore.conf'...
indexing table 'idx1'...
FATAL: failed to open idx1.tmp.spl: Permission denied, will not index. Try --rotate option.
root@ba2345cb4b21:/var/lib/manticore#
```

**Comment by djklim87** [comment]
> @djklim87 Running crontab fails with this error:
> 
> ```
> Manticore 6.2.13 60343e9d9@231017 dev
> Copyright (c) 2001-2016, Andrew Aksyonoff
> Copyright (c) 2008-2016, Sphinx Technologies Inc (http://sphinxsearch.com)
> Copyright (c) 2017-2023, Manticore Software LTD (https://manticoresearch.com)
> 
> using config file '/etc/manticoresearch/manticore.conf'...
> indexing table 'idx1'...
> FATAL: failed to open idx1.tmp.spl: Permission denied, will not index. Try --rotate option.
> root@ba2345cb4b21:/var/lib/manticore#
> ```

fixed in 27d32191eb09041df46379ec4c02f1e281eb5a48

**Comment by PavelShilin89** [comment]
CLT test done - https://github.com/manticoresoftware/docker/actions/runs/6776056919
